packages/tizen_rpc_port/lib/src/stub_base.dart

```diff
@@ -0,0 +1,97 @@
+// You have generated a new plugin project without specifying the `--platforms`
+// flag. A plugin project with no platform support was generated. To add a
+// platform, run `flutter create -t plugin --platforms <platforms> .` under the
+// same directory. You can also find a detailed instruction on how to add
+// platforms in the `pubspec.yaml` at
+// https://flutter.dev/docs/development/packages-and-plugins/developing-packages#plugin-platforms.
+
+import 'dart:async';
+import 'dart:typed_data';
+
+import 'package:tizen_log/tizen_log.dart';
+import 'parcel.dart';
+import 'port.dart';
+import 'rpc_port_method_channel.dart';
+
+const String _logTag = 'RPC_PORT';
+
+Finalizer<StubBase> _finalizer = Finalizer<StubBase>((StubBase stub) {
+  final MethodChannelRpcPort manager = MethodChannelRpcPort.instance;
+  manager.stubDestroy(stub.portName);
+});
+
+/// The base of RpcPort stub class.
+abstract class StubBase {
+  /// The constructor of StubBase.
+  StubBase(this._portName) {
+    final MethodChannelRpcPort manager = MethodChannelRpcPort.instance;
+    manager.stubCreate(_portName);
+    _finalizer.attach(this, this);
+  }
+
+  final String _portName;
+  late final StreamSubscription<dynamic> _streamSubscription;
+  bool _isListening = false;
+
+  /// The port name of connection with proxy.
+  String get portName => _portName;
+
+  /// Sets trusted flag.
+  Future<void> setTrusted(bool trusted) async {
+    final MethodChannelRpcPort manager = MethodChannelRpcPort.instance;
```

Comment:
You may create a private field for this and remove all local variable `manager` in several methods. Also `_methodChannel` seems like a better name.